src/secret_test.js

```diff
@@ -7,7 +7,7 @@ import sinon from "sinon";
 import args from "./arguments.js";
 import * as renderSecret from "./secret.js";
 
-const matches = promisify(renderSecret.matches);
+const matches: (string) => Promise<boolean> = promisify(renderSecret.matches);
```

Comment:
`promisify` is a pain to type. Let's just type what we expect and short-circuit the `any` propagation.